src/ui/handler/drag_rotate.js

```diff
@@ -91,6 +93,14 @@ class DragRotateHandler {
         e.preventDefault();
     }
 
+    /**
+     * Deactivates DragRotate when the window looses focus. Otherwise the mouseup event is never fired and when the
+     * window comes back in focus DragRotate is still active even though the mouse isn't down.
+     */
+    _onBlur(e) {
+        this._onUp(e);
+    }
```

Comment:
Since there's nothing else going on, can we just pass `this._onUp` instead of adding an identical method?